terraform/modules/daos_server/templates/daos_control.yml.tftpl

```diff
@@ -2,6 +2,6 @@
 # DAOS manager (dmg) configuration file
 #
 
-hostlist: ${jsonencode(access_points)}
+hostlist: ${jsonencode(servers)}
```

Comment:
This syntax is coming out a bit wrong in my testing. 

At least in the daos_control.yml  I need to change the syntax to 
"
hostlist: ['daos-server-[0001-0004]']
"
 https://docs.daos.io/v2.0/QSG/setup_centos/#create-configuration-files seems to say all the YML files should use" [' * '] " type syntax.

Reply:
This was fixed on line 41 in terraform/modules/daos_server/main.tf

Reply:
I just tested the current patch. I was able to launch a 4x4 and DMG commands just worked.  The new Syntax is working for me.   Thanks!